index.php

```diff
@@ -1,10 +1,41 @@
 <?php
 
-$db = new PDO(
-    'mysql:host=192.168.20.20, dbname=Guitars',
+$db = new PDO
+(
+    'mysql:host=192.168.20.20; dbname=Guitars',
     'root',
     ''
 );
 
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
+/**
+ *
+ * Returns all desired item values for all items.
+ * Where many images are stored for an item, only the first added (MAIN image) is returned
+ *
+ * @param $database
+ * @return array
+ */
+function getAllFromDatabase($database):array
+{
+    $sqlGetAll = $database->prepare(
+        'SELECT `guitars`.`id`, `images`.`fileLocation`, `brands`.`brand`, `guitars`.`model`, `guitars`.`year`, `types`.`type`,
+                    `countries`.`country`, `guitars`.`value`, `guitars`.`dateAcquired` FROM `guitars`
+            LEFT JOIN `images`
+                ON `images`.`guitarID` = `guitars`.`id`
+            LEFT JOIN `brands`
+                ON `brands`.`id` = `guitars`.`brandID`
+            LEFT JOIN `types`
+                ON `types`.`id` = `guitars`.`typeID`
+            LEFT JOIN `countries`
+                ON `countries`.`id` = `guitars`.`countryID`
+            GROUP BY `guitars`.`id`;'
+    );
+
+    $sqlGetAll->execute();
+    return $sqlGetAll->fetchAll();
+}
+
+$returnAll = getAllFromDatabase($db);
```

Comment:
Naming of $returnAll isn't semantic, $allGuitars instead, maybe?

Reply:
have changed variable to $allGuitars